<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Fortune Level</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fortune Calculator</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FortuneLevel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Fortune Level</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Ore">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Ore</string>
   </property>
   <item>
    <property name="text">
     <string>Coal Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Diamond Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Emerald Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Iron Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Copper Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gold Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nether Gold Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nether Quartz Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lapis Lazuil Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Amethyst Cluster</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Calculate if you mining spectified ore with spectified &lt;/p&gt;&lt;p&gt;fortune level pickaxe repeat spectified times&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CalculateNumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="FortuneProgressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fortune Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ore Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ore</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
